artifactory/commands/mvn/mvn.go

```diff
@@ -140,17 +144,41 @@ func (mc *MvnCommand) CommandName() string {
 	return "rt_maven"
 }
 
+// ConditionalUpload will scan the artifact using Xray and will upload them only if the scan pass with no
+// violation.
 func (mc *MvnCommand) conditionalUpload() error {
```

Comment:
This conditionalUpload methos seems very similar to the Gradle's conditionalUpload. Let's see if we can take their common code into a util and reuse it. We actually have a conditionaluploadutils file. Maybe it's a good place to put the common code.